api/bases/designate.openstack.org_designateagents.yaml

```diff
@@ -0,0 +1,258 @@
+---
```

Comment:
what is the story with all of these new files.
is that a ginkgolinter thing?

Reply:
These are created by `make manifests`: https://github.com/openstack-k8s-operators/designate-operator/blob/5e2c14161b3f1aff5beca3a15fe2293c47d51401/Makefile#L91

Reply:
> what is the story with all of these new files. is that a ginkgolinter thing?

Hey Marios, "pre-commit run -a" command added all the changes.